sensing/autoware_pointcloud_preprocessor/src/downsample_filter/approximate_downsample_filter_node.cpp

```diff
@@ -64,9 +64,9 @@ ApproximateDownsampleFilterComponent::ApproximateDownsampleFilterComponent(
 : Filter("ApproximateDownsampleFilter", options)
 {
   {
-    voxel_size_x_ = static_cast<double>(declare_parameter("voxel_size_x", 0.3));
-    voxel_size_y_ = static_cast<double>(declare_parameter("voxel_size_y", 0.3));
-    voxel_size_z_ = static_cast<double>(declare_parameter("voxel_size_z", 0.1));
+    voxel_size_x_ = declare_parameter<float>("voxel_size_x");
```

Comment:
Are not these doubles in the header?

Reply:
According to this https://github.com/autowarefoundation/autoware.universe/pull/8480#discussion_r1722645727, I changed it to float but forgot to also change the type in the header file!
Thanks for the check! 
fixed in https://github.com/autowarefoundation/autoware.universe/pull/8480/commits/ef88141462ba22bbfa4ae33947b79775d3c6d468